### ISSUE TITLE
Create a new camera helper

### ISSUE THREAD

**Issue by fernandojsg** [aframevr/aframe-inspector#284]
Add a camera helper that looks like a camera or a headset.

**Comment by ngokevin** [comment]
Would also be cool to indicate sitting and standing heights for scale. Like for standing, average height is 1.6 meters, so you can draw a 1.6 meter height wireframe box below the camera.

**Comment by fernandojsg** [comment]
@ngokevin yep good point. Maybe even including something similar to the model that sketchfab uses for the user position (https://help.sketchfab.com/hc/en-us/articles/211640363-VR-Editor)
/cc @feiss

**Comment by feiss** [comment]
Oh, very interesting! I like this concept more than a floating headset. Ok, I could do a sitting and a standing up model. The only drawback I see in this is what happens if the user wants the camera up in the air, or looking up or down, like 45deg down.. The puppet would look a bit weird..

**Comment by ngokevin** [comment]
In the air, I think it still applies. They still have a body and hands, and it helps compare scale.

The head could be separate from the body such that it can rotate freely. The body just follows underneath the camera. Worst case, we can add a toggle in the UI.

**Comment by fernandojsg** [comment]
Yes, I was just thinking the same, it could be hard to manage each situation, when the camera is rotated in a non-standard way. The body typically will be standing or sitting, but won't be laying down on the ground or flying horizontally, so it could help to visualize the user in the world. The body could be fixed (just rotated on Y according to the headset) but the head could be rotated on X/Z

**Comment by feiss** [comment]
That sounds fantastic! Ok, head separated from body. A select for the "camera icon" with options like "standing person, sitting person, camera" would be great indeed

**Comment by fernandojsg** [comment]
Yep, I like that "select"
